controllers/imagehub/controllers/orgclusterrolebinding_controller.go

```diff
@@ -0,0 +1,219 @@
+	db       *DataHelper
+	Scheme   *runtime.Scheme
+	Recorder record.EventRecorder
+}
+
+//+kubebuilder:rbac:groups=imagehub.sealos.io,resources=images,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=imagehub.sealos.io,resources=repositories,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=imagehub.sealos.io,resources=organizations,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=core,resources=namespaces,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=core,resources=serviceaccounts,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterroles/status;roles/status,verbs=get;list;watch
+//+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterroles;roles,verbs=create;delete;get;list;patch;update;watch
+//+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=rolebindings,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterrolebindings,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=imagehub.sealos.io,resources=orgclusterrolebindings,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=imagehub.sealos.io,resources=orgclusterrolebindings/status,verbs=get;update;patch
+//+kubebuilder:rbac:groups=imagehub.sealos.io,resources=orgclusterrolebindings/finalizers,verbs=update
+
+// Reconcile is part of the main kubernetes reconciliation loop which aims to
+// move the current state of the cluster closer to the desired state.
+// TODO(user): Modify the Reconcile function to compare the state specified by
+// the Orgclusterrolebinding object against the actual cluster state, and then
+// perform operations to make the cluster state reflect the state specified by
+// the user.
+//
+// For more details, check Reconcile and its Result here:
+// - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.13.0/pkg/reconcile
+func (r *OrgclusterrolebindingReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	r.Logger.V(1).Info("start reconcile for org")
+
+	org := &imagehubv1.Organization{}
+	ctr := controller.Controller{
+		Client:   r.Client,
+		Logger:   r.Logger,
+		Eventer:  r.Recorder,
+		Operator: r,
+		Gvk: schema.GroupVersionKind{
+			Group:   imagehubv1.GroupVersion.Group,
+			Version: imagehubv1.GroupVersion.Version,
+			Kind:    "Organization",
+		},
+		FinalizerName: imagehubv1.OrgCRBFinalizerName,
+	}
+	org.APIVersion = ctr.Gvk.GroupVersion().String()
+	org.Kind = ctr.Gvk.Kind
+
+	return ctr.Run(ctx, req, org)
+}
+
+func (r *OrgclusterrolebindingReconciler) Update(ctx context.Context, req ctrl.Request, gvk schema.GroupVersionKind, obj client.Object) (ctrl.Result, error) {
+	r.Logger.V(1).Info("update reconcile controller OrgCRB", "request", req)
+	org := &imagehubv1.Organization{}
+	err := convert.JsonConvert(obj, org)
```

Comment:
Why you need to convert org to json? And the obj is unused?

Reply:
refer to user...